bitnami/redis-cluster/templates/svc-cluster-external-access.yaml

```diff
@@ -30,6 +30,9 @@ spec:
   {{- if $root.Values.cluster.externalAccess.service.loadBalancerIP }}
   loadBalancerIP: {{ index $root.Values.cluster.externalAccess.service.loadBalancerIP $i }}
   {{- end }}
+  {{- if and (eq .Values.externalAccess.service.type "LoadBalancer") .Values.externalAccess.service.loadBalancerSourceRanges }}
```

Comment:
Shouldn't these have been `$root.Values.cluster.external...` instead of `.Values.external...`?

Reply:
mmm in fact yes, it seems so... Would you like to contribute by creating a PR to solve the issue? The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md) you can find the contributing guidelines.